jobs/rocoto/awips_20km_1p0deg.sh

```diff
@@ -45,22 +44,17 @@ for fhr3 in ${fhrlst}; do
     fhmax=84
     if (( fhr >= fhmin && fhr <= fhmax )); then
         if ((fhr % 3 == 0)); then
-            export fcsthrs=${fhr3}
-            ${AWIPS20SH}
-        fi
-
-        if ((fhr % 6 == 0)); then
-            ${AWIPSG2SH}
+            export fcsthrs="${fhr3}"
+            "${AWIPS20SH}"
```

Comment:
```suggestion
            "${AWIPS201P0DEGSH}"
```
I would rename this variable in `config.awips`